service/implementations/person_implementation.rb

```diff
@@ -0,0 +1,93 @@
+class PersonImplementation < Hoodoo::Services::Implementation
+
+  # def show( context )
+  #   person = Person.acquire_in!( context )
+  #   return if context.response.halt_processing?
+  #   context.response.set_resources( render_in( context, person ) )
+  # end
```

Comment:
Can you delete this, rather than comment out.  We would look in git history to see previous versions of a file.